### ISSUE TITLE
Picking up from "Electrum" to "Electron Cash" BitcoinCash will not work

### ISSUE THREAD

**Issue by freini** [fyookball/electrum#654]
Hello,
I just wanted to pick up my BitcoinCash, created by the hard fork, via the Wallet Electron Cash.
Now I have the extent that I entered the private key during installation and it works so far.
But I do not get the BitcoinCash !? I synced already but nothing comes.

Can you help me please!??

best regards
Freini

**Comment by marceloneil** [comment]
Did you have bitcoin in that wallet before the fork (before block 478558)?

**Comment by fyookball** [comment]
make sure its not a 2fa wallet and that the same addresses are created.  check the block explorer.

**Comment by gravityguide** [comment]
hi there, - i am attempting to claim bitcoin cash using electron cash wallet- i tried to enter the 12 word seed from my electrum bitcoin (BTC) wallet but the next button did not activate- is there a reason for this? am i doing something wrong? any help would be appreciated- thanks

**Comment by fyookball** [comment]
@gravityguide typo or extra whitespace

**Comment by gravityguide** [comment]
yeah- i have done this - (re: a space inbetween each word- and double checked for typos) the 'next' button only seems to activate roughly half way through typing the 12 word seed (on the 39th character- including spaces), when i finish entering the 12 word seed - the next button does not activate- which is weird...

**Comment by fyookball** [comment]
means one of the words is wrong then.

**Comment by marceloneil** [comment]
Or it could be a BIP39 key, are you sure it's an electrum key?

**Comment by gravityguide** [comment]
yeah -my electrum seed from the electrum BTC wallet is a 12 word seed (total 76 characters)- this is the seed i had my BTC before the fork in august 2017-  this is the seed i am attempting to input into the electron- cash  (BCC) wallet to claim my bitcoin cash

**Comment by marceloneil** [comment]
Was it generated by electrum? You're 100% positive it isn't a BIP39 seed :stuck_out_tongue:?

If it is an electrum key, jonald is right. Typos/bad spacing is likely the issue. Have you tried importing the wallet file directly (obviously make a backup)

**Comment by gravityguide** [comment]
yeah- dont think it is a BIP39 seed- how do i import the wallet file directly- i have the .csv file and json files backed up already- but cant see an option to do this when opening the electron cash wallet- thanks

**Comment by fyookball** [comment]
double check if all the words on are on the seed list.

**Comment by kyuupichan** [comment]
Get the seed words directly from the dialog box in Electrum :  Wallet | Seed menu

**Comment by gravityguide** [comment]
ok - so update:) 
managed to download electron cash - an earlier version  2.9.2
i inputted the electrum (btc) seed ok and it accepted it:) and the (bcc) balance is displayed to claim
however now i cant connect to a server and server not syncing to network
any ideas how i resolve this? thanks

**Comment by fyookball** [comment]
open the wallet file in 3.2

**Comment by gravityguide** [comment]
ok- i downloaded electron cash 3.2
and opened it using the wallet file from (/.electron-cash), however the wallet is still showing 2.9.2 version, but the light in the right corner is showing green and i am now automatically connected to the network (ELECASH.BCC.NUMMI.IT)

so i then attempted to send a transaction to my trezor (BCH) wallet
and got the following error

"error: the transaction was rejected by network rules.
16: mandatory-script-verify-flag-failed(signature must use SIGHASH_FORKID)

still not able to send anything out of my electron cash wallet

**Comment by rt121212121** [comment]
What do you mean the wallet is showing 2.9.2 version?

**Comment by rt121212121** [comment]
@gravityguide, unless you want more help, I am going to close this issue as resolved.  Please respond if you are not sorted out yet.

**Comment by gravityguide** [comment]
still need to look into this further- but thanks for getting back to me...

**Comment by rt121212121** [comment]
The 2.9.2 version and FORKID error indicates that something is really broken on your end, beyond anything possible from a properly installed Electron Cash.  As such, I am going to close this.

Feel free to reopen it when you've investigated, if you think the problem lies with Electron Cash, but given the facts on hand this is very unlikely.
